src/app/components/team/ApiKeyCreate.js

```diff
@@ -0,0 +1,199 @@
+                    helpContent={
+                      <FormattedMessage
+                        id="apiKeyCreate.keyNameHelper"
+                        defaultMessage="The Key Name will be displayed as the author of content added"
+                        description="Label for key name input"
+                      />
+                    }
+                  />
+                )}
+              </FormattedMessage>
+            </div>
+            <p className="typography-subtitle2">
+              <FormattedMessage
+                id="apiKeyCreate.descriptionBlurb"
+                defaultMessage="What will you use this API Key for? (optional)"
+                description="Label displayed before the API Key description input"
+              />
+            </p>
+            <div className={inputStyles['form-fieldset-field']}>
+              <FormattedMessage
+                id="apiKeyCreate.descriptionPlaceholder"
+                defaultMessage="A brief description can help remind the purpose of your integration"
+                description="Placeholder text for api key description input field"
+              >
+                { placeholder => (
+                  <LimitedTextField
+                    className="api-key-create__description-field"
+                    required={false}
+                    maxChars={720}
+                    value={description}
+                    onChange={e => setDescription(e.target.value)}
+                    placeholder={placeholder}
+                    label={
+                      <FormattedMessage
+                        id="global.description"
+                        defaultMessage="Description"
+                        description="Generic label for a text field to input a description"
+                      />
+                    }
+                  />
+                )}
+              </FormattedMessage>
+            </div>
+          </div>
+        </div>
+        <div className={styles['dialog-actions']}>
+          <ButtonMain
+            className="api-key-create__cancel-button"
+            onClick={() => {
+              setDialogOpen(false);
+              resetForm();
+            }}
+            label={
+              <FormattedMessage
+                id="global.cancel"
+                defaultMessage="Cancel"
+                description="Generic label for a button or link for a user to press when they wish to abort an in-progress operation"
+              />
+            }
+            variant="text"
+            theme="text"
+          />
+          <ButtonMain
+            className="api-key-create__submit-button"
+            onClick={handleSubmit}
+            label={
+              <FormattedMessage
+                id="apiKeyCreate.submit"
+                defaultMessage="Generate Key"
+                description="Label for button that creates a new API key"
+              />
+            }
+          />
+        </div>
+      </Dialog>
+    </>
+  );
+};
+
```

Comment:
This is more a personal preference, curious for your thoughts... even when a component doesn't receive any props, I like to declare the empty prop types, just to make it super clear... something like: `ApiKeyCreate.propTypes = {}; // No props`.

Reply:
Yeah, this is better. Will do!